FirebaseStorage/Sources/StorageObservableTask.swift

```diff
@@ -17,18 +17,19 @@ import Foundation
 import FirebaseStorageInternal
 
 /**
- * Extends FIRStorageTask to provide observable semantics such as adding and removing observers.
- * Observers produce a FIRStorageHandle, which is used to keep track of and remove specific
+ * An extended `StorageTask` providing observable semantics that can be used for responding to changes in task state.
```

Comment:
optional but L20 looked a little long. Maybe:
```suggestion
 * An extended `StorageTask` providing observable semantics that can be used for responding to changes 
 * in task state.
```